install_config/upgrading/manual_upgrades.adoc

```diff
@@ -215,7 +280,6 @@ endif::[]
 +
 ----
 # oadm policy reconcile-cluster-role-bindings \
-    --exclude-groups=system:authenticated \
     --exclude-groups=system:authenticated:oauth \
```

Comment:
You can add it to both if you want, it's always been the default for the `reconcile-cluster-role-bindings` command.